assets/dev/scss/frontend/general/_button.scss

```diff
@@ -25,16 +25,23 @@
 	}
 
 	&-icon {
+		i {
```

Comment:
I am not sure what are you doing, but you don't seem to be using the Github interface to see your changes.

This empty line shouldn't have been removed.

@Mchris-hub